src/rpc/protocol/mod.rs

```diff
@@ -209,27 +209,49 @@ impl ProtoDecoder for ObRpcCostTime {
     }
 }
 
-// cluster id
 pub const RPC_PACKET_HEADER_SIZE: usize = HEADER_SIZE + COST_TIME_ENCODE_SIZE //
-                                           + 8 // cluster id
-                                           + 4 // compress type
-                                           + 4 // original len
-                                           ;
+    + 8 // cluster id
+    + 4 // compress type
+    + 4 // original len
+;
+// version 227
+pub const RPC_PACKET_HEADER_SIZE_WITH_NEWSERVER: usize = RPC_PACKET_HEADER_SIZE
```

Comment:
Could the name here be consistent with the [Java client](https://github.com/oceanbase/obkv-table-client-java/blob/master/src/main/java/com/alipay/oceanbase/rpc/protocol/packet/ObRpcPacketHeader.java
)? Also, 
Comments containing "227" need to be removed, the latest observer also has those domains, and comments containing "227" will be confusing.

Reply:
Ok, I will remove comment containing "227", and rename this const variable.

Reply:
how about comment `version: V4`, I refer to java client?